### ISSUE TITLE
Field is_emm_managed of password login details is always expected, but can be missing.

### ISSUE THREAD

**Issue by readshaw** [dropbox/dropbox-sdk-python#113]
I am using the team_log_get_events and team_log_get_events_continue APIs to retrieve user activity for a Dropbox account.  I seem to be able to retrieve most events fine, but I get a validation error from the library for login success events and receive this error:

```
dropbox.stone_validators.ValidationError: events.details.password_login_success_details: missing required field 'is_emm_managed'
```

From http://dropbox-sdk-python.readthedocs.io/en/latest/moduledoc.html#dropbox.team_log.DeviceLogInfo.is_emm_managed, it seems as if this field is normally expected, but could be missing.

Perhaps this validation constraint needs to be relaxed?

**Comment by greg-db** [comment]
Thanks for the report! We'll look into it.

**Comment by readshaw** [comment]
Thanks Greg.  If you need more information, please let me know.

**Comment by readshaw** [comment]
Update: I originally produced this issue with v7.3.1 of the Dropbox python module. 

I received an email today to upgrade to 8.0.0 (unrelated to this issue I assume).  After performing the upgrade to 8.0.0, this is_emm_managed issue remains.

**Comment by greg-db** [comment]
Thanks for the update! That's correct, v8.0.0 addressed a different issue. This one is still open. I'll follow up here once we have an update for it.

**Comment by readshaw** [comment]
Hi @greg-db, any update on this issue, or a suggestion for a workaround?

**Comment by greg-db** [comment]
@readshaw Thanks for the additional note! That's effectively the same issue and should be fixed along with the `is_emm_managed` issue. I don't have a timeline right now though.

**Comment by greg-db** [comment]
This should be fixed by [v8.2.0](https://github.com/dropbox/dropbox-sdk-python/releases/tag/v8.2.0).

**Comment by readshaw** [comment]
Thank you Greg.  I will download 8.2.0 and take a look.
